docs/tempo/website/troubleshooting/_index.md

```diff
@@ -4,152 +4,11 @@ weight: 470
 ---
 
 # Troubleshooting missing traces
-This topic helps with day zero operational issues that may come up when getting started with Tempo. It walks through debugging each part of the ingestion and query pipeline to drill down and diagnose issues.
 
-- [Problem 1. I am unable to see any of my traces in Tempo](#problem-1-i-am-unable-to-see-any-of-my-traces-in-tempo)
-- [Problem 2. Some of my traces are missing in Tempo](#problem-2-some-of-my-traces-are-missing-in-tempo)
-- [Problem 3. Getting error message ‘Too many jobs in the queue’](#problem-3-getting-error-message-too-many-jobs-in-the-queue)
-- [Problem 4. Distributor is not accepting traces](#problem-4-distributor-is-not-accepting-traces)
+The topics in this section helps with day zero operational issues that may come up when getting started with Tempo. They walk you through debugging each part of the ingestion and query pipeline to drill down and diagnose issues.
 
-## Problem 1. I am unable to see any of my traces in Tempo
-**Potential causes**
-- There could be issues in ingestion of the data into Tempo, that is, spans are either not being sent correctly to Tempo or they are not getting sampled.
-- There could be issues querying for traces that have been received by Tempo.
+- [I am unable to see any of my traces in Tempo](unable-to-see-trace)
+- [Some of my traces are missing in Tempo](missing-trace)
+- [I am getting error message ‘Too many jobs in the queue’](too-many-jobs-in-queue)
+- [I have reached the per tenant limit overrides](max-trace-limit-reached)
```

Comment:
```suggestion
- [I have reached the max trace limit](max-trace-limit-reached)
```

Sorry to send you on the wrong path, the original message made sense here. Re-adding with this suggestion.